docs/conf.py

```diff
@@ -49,9 +49,12 @@
 # intersphinx mapping
 intersphinx_mapping = {
     'numpy': ('https://numpy.org/doc/stable/', None),
+    'numpydoc': ('https://numpydoc.readthedocs.io/en/latest/', None),
     'scipy': ('https://docs.scipy.org/doc/scipy/', None),
     'matplotlib': ('https://matplotlib.org/stable/', None),
-    'spharpy': ('https://spharpy.readthedocs.io/en/stable/', None)
+    'spharpy': ('https://spharpy.readthedocs.io/en/stable/', None),
+    'pytest': ('https://docs.pytest.org/en/stable/', None),
+    'coverage': ('https://coverage.readthedocs.io/en/5.5/', None),
```

Comment:
Why link to v5.5? It seems that coverage uses `latest` to link to the current stable version
```suggestion
    'coverage': ('https://coverage.readthedocs.io/en/latest/', None),
```

Reply:
this was the link before, but i think we can switch to the latest version